lib/rubocop/cop/minitest/assert_raises_with_regexp_argument.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Minitest
+      # Checks for `assert_raises` with argument of regular expression literal.
+      # Arguments should be exceptions.
```

Comment:
How about adding details like this?

```suggestion
      # Arguments should be exception classes.
      # Or that argument takes an optional custom message on the end to help explain failures.
      # The message should be a string. Either way, it's not the argument that `exception.message` is compared to.
```